pkg/transform/oauth_transform.go

```diff
@@ -55,6 +56,17 @@ func (e OAuthExtraction) Transform() (Output, error) {
 			m := Manifest{Name: filename, CRD: secretCR}
 			manifests = append(manifests, m)
 		}
+
```

Comment:
I'm thinking of a way to avoid repeting L59-69 in golang, they are similiar to L49-58

Reply:
Not sure if it worth creating a Manifest builder.

Reply:
If you see this crop up again, that may be time to introduce something like a generic interface:

```
type ManifestGenerator interface {
  GenYAML() string
}
```

And have one function handle this. Rule of 3 is useful, if you see code like this show up 3 times, it's a pretty good indication it's time to refactor to something more generic because there's obviously shared behavior in there somewhere.